src/osip-opcua-server/src/main/java/edu/kit/pse/osip/core/opcua/server/UANamespaceWrapper.java

```diff
@@ -94,6 +103,9 @@ protected void addVariable(String path, String displayName, NodeId type) {
         variableNodes.put(path, newNode);
 
         String folderPath = path.substring(0, path.lastIndexOf("/"));
+        if (!folderNodes.containsKey(folderPath)) {
+            throw new IllegalStateException("Parent folder not found");
+        }
         folderNodes.get(folderPath).addOrganizes(newNode);
     }
 
```

Comment:
Da kann ich nichts kommentieren: In Zeile 133 (neu) "to" ohne zweites t und in Zeile 135 (neu) "browse" mit r.